feat: Add release details docs (#3907)
getsentry/sentry-docs: src/docs/product/releases/health/release-details.mdx

description: "Learn more about the details of individual releases."
---

The **Release Details** page focuses on an individual release. Elements of the release are presented in visual trends for crashes and sessions, specifics regarding each issue, adoption graphs, and commit author breakdowns.

> ![View of the release details page showing a release comparison graph, issues, adoption, and other details.](release_details.png)

Reviewer: It seems like all of these features that you're describing are early adopter features. If so, and if you're hoping to merge this before those features go GA, then you'll need an EA note:

```suggestion
![View of the release details page showing a release comparison graph, issues, adoption, and other details.](release_details.png)

<Note>

The following features are available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

```

## Overview
### Graphs

> After selecting the time range (the default is the entire release period), the top chart provide insight into:

Reviewer: ```suggestion

The graph at the top of the page provides insights into health of your release by allowing you to choose which metric it displays. Select the time range that you want to review (the default range is the entire release period) and choose from these metrics in the table below the graph:
```

- [Crash free session](/product/releases/health/#crash-free-sessionsusers) rate
- [Healthy](/product/releases/health/#healthy), [abnormal](/product/releases/health/#abnormal), [errored](/product/releases/health/#errored) session rate
- [Crashed session](/product/releases/health/#crashed-users) rate
- [Failure rate](/product/performance/metrics/#failure-rate)
- [Session](/product/releases/health/#session) count
- Error count
- Transaction count

> Percentage based metrics of this release are compared to average of all releases to easily spot the difference in trend.

Reviewer: ```suggestion
In the table below the graph, percentage-based metrics of this release are compared to average of all releases so you can spot any differences in trend.
```

> [Errored](/product/releases/health/#errored) sessions most likely result in handled issues and [crashed](/product/releases/health/#crash) sessions in unhandled issues. Click on the handled/unhandled issues link to see them in [Issues](/product/issues/) page.

Reviewer: ```suggestion
[Errored](/product/releases/health/#errored) sessions are likely to result in handled issues, and [crashed](/product/releases/health/#crash) sessions in unhandled issues. Click on the handled/unhandled issues link to see them in [Issues](/product/issues/) page.
```

> Click on any metric to display it in the top chart.
> 

Reviewer: ```suggestion
```

> Click on any series in the chart legend to hide it (hide Healthy series to see Crashes in better scale).

Reviewer: ```suggestion
Click on any series in the chart legend to hide it. For example, you might hide the Healthy series to see Crashes in better scale.
```

Hover over specific dates and times in the overview graph for a quick perspective of events that crashed, behaved abnormally, errored, or are healthy, as defined for [Release Health](/product/releases/health/).

> ### Issues

Reviewer: include spaces before and after headings

```suggestion

## Issues

```

> Use Release Details to view issues associated with this release and to access them for triage. You can navigate to the [Issues](/product/issues/) page by clicking Open in Issues button located at the top of the issues table. The issues in the table are sorted by the number of events in the selected time range. Click the filter buttons to see:

Reviewer: ```suggestion
Use the **Release Details** page to view issues associated with this release and to access them for triage. You can navigate to the [Issues](/product/issues/) page by clicking the "Open in Issues" button located at the top of the issues table. The issues in the table are sorted by the number of events in the selected time range. Click the filter buttons to see:
```

- All Issues
- New Issues
- Unhandled Issues
- Resolved Issues 

> ### Sessions Adopted

Reviewer: ```suggestion
## Sessions Adopted

```

> See the release adoption percentage over time. This graph compares sessions on this release with session on all releases. You can change main Date Range to see adoption in the different timeframe.

Reviewer: ```suggestion
See the release adoption percentage over time. This graph compares sessions on this release with sessions on all releases. You can change main date range to see adoption in a different timeframe.
```

> ### Commit Author Breakdown

Reviewer: ```suggestion
## Commit Author Breakdown

```

See the list of people that authored commits in this release together with the absolute and relative counts of commits per person. [Learn how to set up commits](/product/releases/suspect-commits/).

> ### Deploys

Reviewer: ```suggestion
## Deploys

```

> See when and to what environments this release has been deployed to. Click on the deploy badge to see issues from that environment. [Learn how to set up deploys](/product/releases/setup/#notify-sentry).

Reviewer: ```suggestion
See when and to which environments this release has been deployed. Click on the deploy badge to see issues from that environment. [Learn how to set up deploys](/product/releases/setup/#notify-sentry).
```

> ### Other Projects in This Release

Reviewer: ```suggestion
## Other Projects in This Release

```

> Releases are global per organization. See here the list of projects with the same release version. Click on project to go to [Project Details](/product/projects/project-details/), click on View to see details of that project's release.

Reviewer: ```suggestion
Releases are global per organization. This list shows you other projects with the same release version. Click on project to go to [Project Details](/product/projects/project-details/), and click "View" to see details of that project's release.
```

description: "Learn more about the details of individual releases."
---

The **Release Details** page focuses on an individual release. Elements of the release are presented in visual trends for crashes and sessions, specifics regarding each issue, adoption graphs, and commit author breakdowns.

![View of the release details page showing a release comparison graph, issues, adoption, and other details.](release_details.png)

<Note>

The following features are available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

## Graphs

> The graph at the top of the page provides insights into health of your release by allowing you to choose which metric it displays. Select the time range that you want to review (the default range is the entire release period) and choose from these metrics in the table below the graph:

Reviewer: ```suggestion
The graph at the top of the page provides insights into the health of your release by allowing you to choose which metric it displays. Select the time range that you want to review (the default range is the entire release period) and choose from these metrics in the table below the graph:
```

- [Crash free session](/product/releases/health/#crash-free-sessionsusers) rate
- [Healthy](/product/releases/health/#healthy), [abnormal](/product/releases/health/#abnormal), [errored](/product/releases/health/#errored) session rate
- [Crashed session](/product/releases/health/#crashed-users) rate
- [Failure rate](/product/performance/metrics/#failure-rate)
- [Session](/product/releases/health/#session) count
- Error count
- Transaction count

> In the table below the graph, percentage-based metrics of this release are compared to average of all releases so you can spot any differences in trend.

Reviewer: ```suggestion
In the table below the graph, percentage-based metrics of this release are compared to the average of all releases so you can spot any differences in trend.
```

[Errored](/product/releases/health/#errored) sessions are likely to result in handled issues, and [crashed](/product/releases/health/#crash) sessions in unhandled issues. Click on the handled/unhandled issues link to see them in [Issues](/product/issues/) page.

> Click on any series in the chart legend to hide it. For example, you might hide the Healthy series to see Crashes in better scale.

Reviewer: "Series" might not be clear to users. Have tried something else here

```suggestion
Click on any option in the graph legend to hide it. For example, you might hide Healthy sessions to see Crashes in better scale.
```

Hover over specific dates and times in the overview graph for a quick perspective of events that crashed, behaved abnormally, errored, or are healthy, as defined for [Release Health](/product/releases/health/).

## Issues

Use the **Release Details** page to view issues associated with this release and to access them for triage. You can navigate to the [Issues](/product/issues/) page by clicking the "Open in Issues" button located at the top of the issues table. The issues in the table are sorted by the number of events in the selected time range. Click the filter buttons to see:
- All Issues
- New Issues
- Unhandled Issues
- Resolved Issues 

## Sessions Adopted

> See the release adoption percentage over time. This graph compares sessions on this release with sessions on all releases. You can change main date range to see adoption in a different timeframe.

Reviewer: ```suggestion
See the release adoption percentage over time. This graph compares sessions on this release with sessions on all releases. You can change the main date range to see adoption in a different timeframe.
```

## Commit Author Breakdown

See the list of people that authored commits in this release together with the absolute and relative counts of commits per person. [Learn how to set up commits](/product/releases/suspect-commits/).

## Deploys

See when and to which environments this release has been deployed. Click on the deploy badge to see issues from that environment. [Learn how to set up deploys](/product/releases/setup/#notify-sentry).

## Other Projects in This Release

> Releases are global per organization. This list shows you other projects with the same release version. Click on project to go to [Project Details](/product/projects/project-details/), and click "View" to see details of that project's release.

Reviewer: ```suggestion
Releases are global per organization, and this list shows you other projects with the same release version. Click on project to go to [Project Details](/product/projects/project-details/), and click "View" to see details of that project's release.
```
